### ISSUE TITLE
Implementation of X

### ISSUE THREAD

**Issue by tony-ward** [Lora-net/LoRaMac-node#1278]

**Comment by vtjballeng** [comment]
Indeed, we are also seeing this sensitivity errata continue when this is only set at startup with a ~6dBm variation between 250kHz & 500kHz. 

We've validated the solution is to call this for each tx event which brings 500kHz to the expected ~3dBm difference to 250kHz.

**Comment by mluis1** [comment]
Sorry for the late comment on this issue.

If I am not mistaken the stack always call `RadioSetTxConfig` prior to setting the radio in Tx mode. Thus, we think that before every uplink the workaround is applied.

Can you please indicate a situation where this is not the case?
